.dassie/config/role_map.yml

```diff
@@ -24,3 +24,4 @@ test:
 production:
   admin:
     - admin@example.com
+    - another_admin@example.com
```

Comment:
this and the related seed change could go in separately and right away, if it helps.

Reply:
i took this out.